manifests/b/BrightSign/BSNEE/1.02/BrightSign.BSNEE.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BrightSign.BSNEE

```

Comment:
Can you explain the choice for the PackageIdentifier? I'm confused as to why it wouldn't be `BrightSign.BrightAuthorConfigurator`

Reply:
I don’t want to rename it because you can’t rename folders or move multiple items on GitHub yet.

Reply:
> I don’t want to rename it because you can’t rename folders or move multiple items on GitHub yet.

Pressing the period (`.`) while on the files changed view will open VsCode online which allows you to rename the folders

Reply:
I shortened it due to it being too long beforehand.

Reply:
> > I don’t want to rename it because you can’t rename folders or move multiple items on GitHub yet.
> 
> Pressing the period (`.`) while on the files changed view will open VsCode online which allows you to rename the folders

I currently do not own a device capable of running VSCode, as my laptops HHD is 5400 RPM and cannot be upgraded at this time.

Reply:
This would be the website version of VSCode, for which HDD speed should hopefully be less relevant. It shouldn't be much more demanding than this webpage.